code/act_comm.c

```diff
@@ -2660,17 +2629,22 @@ void perm_death_log(CHAR_DATA *ch, int deltype)
 		return;
 
 	fprintf(fp, "%s||[%2d %5s %s]||%s%s%s%s||Kills: %d||Deaths: %d||Age: %d||Hours: %d||%s",
```

Comment:
Hours: `%d`

should probably be

Hours: `%ld` 
(format for signed long)

or

Hours `%lu` 
(format for unsigned long)

Reply:
I didn't know that, I'll look up the formatting rules and adjust. Thanks!